src/PanoViewer/PanoViewer.js

```diff
@@ -57,9 +58,20 @@ export default class PanoViewer extends Component {
 			return this;
 		}
 
+		if (!!options.image && !!options.video) {
+			setTimeout(() => {
+				this.trigger(EVENTS.ERROR, {
+					type: ERROR_TYPE.INVALID_RESOURCE,
+					message: "Specifying multi resouces(both image and video) is not valid."
+				});
+			}, 0);
+			return this;
+		}
+
 		this._container = container;
-		this._image = options.image;
-		this._imageType = options.imageType || PanoImageRenderer.ImageType.EQUIRECTANGULAR;
+		this._image = options.image || options.video;
+		this._isVideo = !!options.video;
+		this._projectionType = options.projectionType || PanoImageRenderer.ImageType.EQUIRECTANGULAR;
```

Comment:
@happyhj `_imageType` -> `_projectionType` on PanoViewer. It is not applied on PanoImageRenderer